registry/pom.xml

```diff
@@ -93,8 +89,8 @@
 			<version>${terrakube-client-starter.version}</version>
 			<exclusions>
 				<exclusion>
-					<groupId>com.squareup.okio:okio-jvm</groupId>
-					<artifactId>3.0.0 </artifactId>
+					<groupId>com.squareup.okio</groupId>
```

Comment:
Looks like it was not working initially and when I fixed that tests start to fail. Removing that exclusion will fix the build. @alfespa17 what will be your suggestion here?

Reply:
I am not sure, this is why I never fixed the warning messages

Reply:
I just double checked and it was not working. You can test it by doing that:
- check this branch (including proper exclusion):
```
cd registry
mvn help:effective-pom
```
- check main branch:
```
cd registry
mvn help:effective-pom
```

You will see that in the main branch effective pom does not contains that exclusion. In other words we should remove it completely.

Reply:
Re-based with latest changes related to sonar build.